boskos/janitor/deployment.yaml

```diff
@@ -61,4 +61,24 @@ spec:
       - name: service
         secret:
           secretName: service-account
-
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: boskos-janitor-aws
+  labels:
+    app: boskos-janitor-aws
+  namespace: test-pods
+spec:
+  replicas: 4  # 4 distributed janitor instances
+  template:
+    metadata:
+      labels:
+        app: boskos-janitor-aws
+    spec:
+      terminationGracePeriodSeconds: 300
+      containers:
+      - name: boskos-janitor-aws
+        image: gcr.io/k8s-testimages/janitor-aws:v20180725-c421ed4f9
```

Comment:
this should be the new `aws-janitor-boskos` image

Reply:
Hi @krzyzacy,

Did you call the old image `janitor-aws`? If you look at the `Makefile` changes, the image and program name are both configurable. I just used the most succinct addition to the root of `janitor` to connote the new type.

Reply:
I think I need to help you push a new image regardless when you are ready :-)

Reply:
Hi @krzyzacy,

Yup. I have no idea how the image tags are handled. I figure there is a process for that. Especially as I cannot push anyway :)

Reply:
Hi @krzyzacy,

> gcr.io/k8s-test-infra-aws/aws-janitor

I think it's safe to call the new image `gcr.io/k8s-testimages/janitor-aws` since the old periodic image uses a different org and a different name. The new image follows the same naming convention as the one used with the root janitor service, and this one is for AWS, hence the suffix makes sense.

Reply:
Hi @krzyzacy,

I force pushed all the suggested changes, but I did leave the name of the image as `gcr.io/k8s-testimages/janitor-aws`. To me it falls in line with the existing Boskos janitor image name. Please let me know if you still want me to change it.

Reply:
Hi @krzyzacy,

For the time being I modified the pod's container to use the image `akutz/boskos-janitor-aws:v20190226-99e2515db`, which is just the image pushed to my personal Docker repo. It should allow for testing.